modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx

```diff
@@ -21,6 +22,13 @@ import openDefaultFailureToast from '../../utils/openDefaultFailureToast';
 import openDefaultSuccessToast from '../../utils/openDefaultSuccessToast';
 import {IFDSAction} from '../Actions';
 
+const ACTION_METHOD = {
+	DELETE: 'DELETE',
+	GET: 'GET',
+	PATCH: 'PATCH',
+	POST: 'POST',
+};
+
```

Comment:
(optional) You can consider `enum`s for this type of usage, which may be more descriptive than simple objects. We started adding them in [few places in DSM](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/types.ts#L8-L18). It's optional, simple objects is fine as well.